README.md

```diff
@@ -1,11 +1,9 @@
-# OmniAuth::MultiProvider
+# OmniAuth MultiProvider
 
-Welcome to your new gem! In this directory, you'll find the files you need to be
-able to package up your Ruby library into a gem. Put your Ruby code in the file
-`lib/omniauth-multi-provider`. To experiment with that code, run 
-`bin/console` for an interactive prompt.
-
-TODO: Delete this and the text above, and describe your gem
+This is a simple extension to [omniauth](https://github.com/omniauth/omniauth) for supporting 
+multiple identity provider instances of a given type e.g. multiple SAML 
+identity providers. It is a generalization of the 
+[omniauth-multi-provider-saml](https://github.com/salsify/omniauth-multi-provider-saml).
```

Comment:
Is there going to be notice on this repo that the gem is deprecated and this generalization is being maintained instead?